helm_deploy/prisoner-content-hub-backend/readme.md

```diff
@@ -36,7 +36,8 @@ helm upgrade [Release Name] . \
 --namespace [Kubernetes Namespace] \
 --values values.[Environment].yaml \
 --values secrets.yaml \
---set image.tag=[Image Tag]
+--set image.tag=[Image Tag] \
+--set cronToken=[Drupal cron token]
```

Comment:
Would it be worth adding where to find the Drupal cron token (Configuration -> Cron)?